.github/actions/common-setup-ec2-linux/action.yml

```diff
@@ -0,0 +1,23 @@
+name: Setup EC2 Linux and workspace
+
+description: Setup EC2 Linux and workspace
+
+inputs:
+  GITHUB_TOKEN:
+    required: true
+
+runs:
+  using: composite
+  steps:
+    - name: Setup Linux
```

Comment:
What is the difference between this action and `setup-linux`? The name and description are almost identical. When should I use one or the other?

Reply:
I need to improve the name of this action...or maybe inline it. This one calls setup linux and does a bit more stuff afterwards